dist/themes/cyverse/templates/main.css

```diff
@@ -33,12 +33,12 @@ article img {
     width: 100%;
 }
 
-code {
+pre code {
     line-height: 1.2;
     display: block;
     overFlow: scroll;
     padding: 10px;
-    /* 
+    /*
```

Comment:
We can remove the rest of the properties in this selector's block because they are being applied in the `code` selector below. This comment should then be moved to the corresponding `font-size` rule on line 52.